daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -492,7 +512,7 @@ func (devices *DeviceSet) createFilesystem(info *DevInfo) error {
 	return nil
 }
 
-func (devices *DeviceSet) migrateOldMetaData() error {
+func (devices *DeviceSet) migrateOldmetaData() error {
```

Comment:
It's probably a nit but feels weird, I would tend to prefer ``migrateOldMetadata`` than ``migrateOldemetaData`` as here the camel case is there to separate words (and thus it *reads* "migrate oldmeta data" instead of "migrate old metadata").